### ISSUE TITLE
Error after upgrade from 2.8.4 to 2.8.5

### ISSUE THREAD

**Issue by bastelfreak** [fomantic/Fomantic-UI-SASS#8]
Hi,
we use your gem at https://github.com/voxpupuli/vox-pupuli-tasks.
After updating your gem from 2.8.4 to 2.8.5 we get the following error while building the application:

```
bastelfreak@bastelfreak-ws ~/vox-pupuli-tasks $ SECRET_KEY_BASE=$(bundle exec rails secret) bundle exec rails assets:precompile
DEPRECATION WARNING: Single arity template handlers are deprecated. Template handlers must
now accept two parameters, the view object and the source for the view object.
Change:
  >> Coffee::Rails::TemplateHandler.call(template)
To:
  >> Coffee::Rails::TemplateHandler.call(template, source)
 (called from <main> at /home/bastelfreak/vox-pupuli-tasks/Rakefile:8)
I, [2020-07-22T20:03:18.095213 #63546]  INFO -- sentry: ** [Raven] Raven 2.13.0 configured not to capture errors: DSN not set
DEPRECATION WARNING: Single arity template handlers are deprecated. Template handlers must
now accept two parameters, the view object and the source for the view object.
Change:
  >> Coffee::Rails::TemplateHandler.call(template)
To:
  >> Coffee::Rails::TemplateHandler.call(template, source)
 (called from <main> at /home/bastelfreak/vox-pupuli-tasks/Rakefile:8)
I, [2020-07-22T20:03:19.418046 #63546]  INFO -- : Writing /home/bastelfreak/vox-pupuli-tasks/public/assets/application-eee7b95cec99d522f7655bd301f498b5e914b4cf5b3eef837c17d8bb3570a03b.js
I, [2020-07-22T20:03:19.418470 #63546]  INFO -- : Writing /home/bastelfreak/vox-pupuli-tasks/public/assets/application-eee7b95cec99d522f7655bd301f498b5e914b4cf5b3eef837c17d8bb3570a03b.js.gz
rails aborted!
Sass::SyntaxError: Invalid CSS after "i.icon.stopwatch.": expected class name, was "20:before"
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/fomantic-ui-sass-2.8.5/app/assets/stylesheets/semantic-ui/elements/_icon.scss:3041
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/fomantic-ui-sass-2.8.5/app/assets/stylesheets/semantic-ui/elements/_all.scss:7
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/fomantic-ui-sass-2.8.5/app/assets/stylesheets/semantic-ui.scss:2
/home/bastelfreak/vox-pupuli-tasks/app/assets/stylesheets/application.scss:17
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/scss/parser.rb:1308:in `expected'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/scss/parser.rb:1238:in `expected'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/scss/parser.rb:1233:in `tok!'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/scss/parser.rb:1097:in `ident!'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/scss/static_parser.rb:194:in `class_selector'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/scss/static_parser.rb:162:in `simple_selector_sequence'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/scss/static_parser.rb:108:in `selector'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/scss/static_parser.rb:83:in `selector_comma_sequence'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/scss/parser.rb:1216:in `expr!'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/scss/static_parser.rb:20:in `parse_selector'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:434:in `visit_rule'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/base.rb:36:in `visit'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:158:in `block in visit'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/stack.rb:79:in `block in with_base'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/stack.rb:135:in `with_frame'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/stack.rb:79:in `with_base'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:158:in `visit'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:325:in `block (2 levels) in visit_import'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:325:in `map'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:325:in `block in visit_import'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/stack.rb:88:in `block in with_import'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/stack.rb:135:in `with_frame'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/stack.rb:88:in `with_import'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:322:in `visit_import'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/base.rb:36:in `visit'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:158:in `block in visit'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/stack.rb:79:in `block in with_base'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/stack.rb:135:in `with_frame'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/stack.rb:79:in `with_base'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:158:in `visit'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:325:in `block (2 levels) in visit_import'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:325:in `map'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:325:in `block in visit_import'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/stack.rb:88:in `block in with_import'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/stack.rb:135:in `with_frame'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/stack.rb:88:in `with_import'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:322:in `visit_import'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/base.rb:36:in `visit'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:158:in `block in visit'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/stack.rb:79:in `block in with_base'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/stack.rb:135:in `with_frame'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/stack.rb:79:in `with_base'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:158:in `visit'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:325:in `block (2 levels) in visit_import'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:325:in `map'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:325:in `block in visit_import'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/stack.rb:88:in `block in with_import'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/stack.rb:135:in `with_frame'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/stack.rb:88:in `with_import'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:322:in `visit_import'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/base.rb:36:in `visit'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:158:in `block in visit'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/stack.rb:79:in `block in with_base'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/stack.rb:135:in `with_frame'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/stack.rb:79:in `with_base'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:158:in `visit'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/base.rb:52:in `block in visit_children'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/base.rb:52:in `map'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/base.rb:52:in `visit_children'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:167:in `block in visit_children'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:179:in `with_environment'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:166:in `visit_children'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/base.rb:36:in `block in visit'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:186:in `visit_root'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/base.rb:36:in `visit'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:157:in `visit'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:10:in `visit'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/root_node.rb:36:in `css_tree'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/tree/root_node.rb:20:in `render'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-3.7.4/lib/sass/engine.rb:290:in `render'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sass-rails-5.1.0/lib/sass/rails/template.rb:47:in `evaluate'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/tilt-2.0.10/lib/tilt/template.rb:109:in `render'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/legacy_tilt_processor.rb:25:in `call'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/processor_utils.rb:75:in `call_processor'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/processor_utils.rb:57:in `block in call_processors'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/processor_utils.rb:56:in `reverse_each'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/processor_utils.rb:56:in `call_processors'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/loader.rb:134:in `load_from_unloaded'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/loader.rb:60:in `block in load'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/loader.rb:317:in `fetch_asset_from_dependency_cache'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/loader.rb:44:in `load'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/cached_environment.rb:20:in `block in initialize'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/cached_environment.rb:47:in `load'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/bundle.rb:23:in `block in call'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/utils.rb:200:in `dfs'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/bundle.rb:24:in `call'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/processor_utils.rb:75:in `call_processor'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/processor_utils.rb:57:in `block in call_processors'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/processor_utils.rb:56:in `reverse_each'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/processor_utils.rb:56:in `call_processors'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/loader.rb:134:in `load_from_unloaded'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/loader.rb:60:in `block in load'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/loader.rb:317:in `fetch_asset_from_dependency_cache'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/loader.rb:44:in `load'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/cached_environment.rb:20:in `block in initialize'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/cached_environment.rb:47:in `load'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/base.rb:66:in `find_asset'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/base.rb:73:in `find_all_linked_assets'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/manifest.rb:142:in `block in find'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/legacy.rb:114:in `block (2 levels) in logical_paths'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/path_utils.rb:228:in `block in stat_tree'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/path_utils.rb:212:in `block in stat_directory'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/path_utils.rb:209:in `each'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/path_utils.rb:209:in `stat_directory'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/path_utils.rb:227:in `stat_tree'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/legacy.rb:105:in `each'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/legacy.rb:105:in `block in logical_paths'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/legacy.rb:104:in `each'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/legacy.rb:104:in `logical_paths'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/manifest.rb:140:in `find'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/sprockets/manifest.rb:186:in `compile'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-rails-3.2.1/lib/sprockets/rails/task.rb:68:in `block (3 levels) in define'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-3.7.2/lib/rake/sprocketstask.rb:147:in `with_logger'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/sprockets-rails-3.2.1/lib/sprockets/rails/task.rb:67:in `block (2 levels) in define'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/railties-6.0.3.2/lib/rails/commands/rake/rake_command.rb:23:in `block in perform'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/railties-6.0.3.2/lib/rails/commands/rake/rake_command.rb:20:in `perform'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/railties-6.0.3.2/lib/rails/command.rb:48:in `invoke'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/railties-6.0.3.2/lib/rails/commands.rb:18:in `<main>'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/activesupport-6.0.3.2/lib/active_support/dependencies.rb:324:in `block in require'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/activesupport-6.0.3.2/lib/active_support/dependencies.rb:291:in `load_dependency'
/home/bastelfreak/vox-pupuli-tasks/vendor/ruby/2.7.0/gems/activesupport-6.0.3.2/lib/active_support/dependencies.rb:324:in `require'
bin/rails:6:in `<main>'
Tasks: TOP => assets:precompile
(See full trace by running task with --trace)
bastelfreak@bastelfreak-ws ~/vox-pupuli-tasks $
```

this seems to be caused by the following change:

https://github.com/fomantic/Fomantic-UI-SASS/compare/v2.8.4...v2.8.5#diff-726b9e44df36f7022d197285d1eec305R3041

if I understand this correctly, the file is vendored from Fomantic-UI. It seems to be fixed, but not yet released: https://github.com/fomantic/Fomantic-UI/issues/1544

**Comment by lubber-de** [comment]
This was fixed in fomantic-ui 2.8.7 by https://github.com/fomantic/Fomantic-UI/pull/1546
This port just needs an update 😉 

Same issue mentioned in #11 and #12

**Comment by shanecav84** [comment]
Thanks again @lubber-de. I've release 2.8.7 which hopefully fixes the issue
